spec/features/test_page_spec.rb

```diff
@@ -4,3 +4,28 @@
     expect(page).to have_content('test page')
   end
 end
+
+# As a marketeer
+# So that I can see my name in lights
+# I would like to register my name before playing an online game
+
+feature 'player name' do
+  scenario 'player can enter their name' do
+    fill_in_name
+    expect(page).to have_content('Mike! Choose your weapon:')
+  end
+end
+
+# As a marketeer
+# So that I can enjoy myself away from the daily grind
+# I would like to be able to play rock/paper/scissors
+
+feature 'allows play' do
+  scenario 'player can play rock/paper/scissors against the computer' do
+    visit('/game')
+    page.find('#rock').click
+    click_button 'Go!'
+    allow_any_instance_of(RockPaperScissors).to receive(:computer_weapon).and_return(:Scissors)
+    expect(page).to have_content('You win!')
```

Comment:
not sure this test 100% covers the behaviour you're trying to test for, in that it's actually testing that you win is displayed but not that it's registering the chosen options and carrying out any logic to dynamically produce a winner. You're restricting yourself by trying to cover a lot of ground in one test - perhaps consider breaking the behaviour down into multiple steps you can test for?